go/libraries/doltcore/migrate/progress.go

```diff
@@ -31,37 +37,34 @@ import (
 	"github.com/dolthub/dolt/go/store/val"
 )
 
-type ChunkMapping interface {
-	Has(ctx context.Context, addr hash.Hash) (bool, error)
-	Get(ctx context.Context, addr hash.Hash) (hash.Hash, error)
-	Put(ctx context.Context, old, new hash.Hash) error
-	Close(ctx context.Context) error
-}
-
-type CommitStack interface {
-	Push(ctx context.Context, cm *doltdb.Commit) error
-	Pop(ctx context.Context) (*doltdb.Commit, error)
-}
+const (
+	migratedCommitsBranch = "migrated_commits"
+	migratedCommitsTable  = "mapping"
```

Comment:
Use a reserved table and branch name for this

`dolt_migrated_commits`
`dolt_commit_mapping`